internal/flags/helpers.go

```diff
@@ -239,15 +239,24 @@ func AutoEnvVars(flags []cli.Flag, prefix string) {
 		case *cli.StringFlag:
 			flag.EnvVars = append(flag.EnvVars, envvar)
 
+		case *cli.StringSliceFlag:
+			flag.EnvVars = append(flag.EnvVars, envvar)
+
 		case *cli.BoolFlag:
 			flag.EnvVars = append(flag.EnvVars, envvar)
 
 		case *cli.IntFlag:
 			flag.EnvVars = append(flag.EnvVars, envvar)
 
+		case *cli.Int64Flag:
+			flag.EnvVars = append(flag.EnvVars, envvar)
+
 		case *cli.Uint64Flag:
 			flag.EnvVars = append(flag.EnvVars, envvar)
 
+		case *cli.Float64Flag:
+			flag.EnvVars = append(flag.EnvVars, envvar)
+
 		case *cli.DurationFlag:
 			flag.EnvVars = append(flag.EnvVars, envvar)
```

Comment:
I'm late to this review, but just an offer that this could be written more compactly/without duplication:

```
switch flat := flag.(type) {
case
    *cli.BoolFlag,
    *cli.IntFlag,
    *cli.Int64Flag,
    *cli.Uint64Flag,
    *cli.Float64Flag,
    *cli.DurationFlag:
        flag.EnvVars = append(flag.EnvVars, envvar)
}
```

Reply:
very good idea, let's open a new PR to fix this :)